script.js

```diff
@@ -16,3 +16,320 @@ navItem.forEach((list) => {
+  </div>
+  </div>
+  </div>
+  <div>
+  `;
+  projectContainer.innerHTML += eachProject;
+});
+
+// Render Desktop
+const desktopProjects = [
+  {
+    name: 'Multi Post Stories',
+    technologies: ['Facebook', 'Fullstack Dev', '2015'],
+    information:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desktopImages: 'desktop/1-Snapshoot-Portfolio.png',
+  },
+  {
+    name: 'Multi Post Stories',
+    technologies: ['Facebook', 'Fullstack Dev', '2015'],
+    information:
+    'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desktopImages: 'desktop/Avail-Snapshoot-Portfolio.png',
+  },
+  {
+    name: 'Multi Post Stories',
+    technologies: ['Facebook', 'Fullstack Dev', '2015'],
+    information:
+    'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desktopImages: 'desktop/Snapshoot-Portfolio.png',
+  },
+  {
+    name: 'Multi Post Stories',
+    technologies: ['Facebook', 'Fullstack Dev', '2015'],
+    information:
+    'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desktopImages: 'desktop/Print-Snapshoot-Portfolio.png',
+  },
+];
+
+desktopProjects.forEach((desktopProject, desktopIndex) => {
+  let technology = '';
+  desktopProject.technologies.forEach((techItem) => {
+    technology += ` <li class="canopy">${techItem}</li>`;
+  });
+  let language = '';
+  desktopProject.languages.forEach((lang) => {
+    language += `<li class="programming-languages"><span>${lang}</span></li>`;
+  });
+
+  const eachDesktopProject = `
+  <div class = "desktop-cards">
+  <div class="project-card">
+  <img
+  class="snapshot-portfolio-img"
+    src=${desktopProject.desktopImages}
+    alt="snapshot-portfolio"
+  />
+  <div class="project-text">
+    <h2 class="project-title">${desktopProject.name}</h2>
+    <div class="counter">
+    ${technology}
+    </div>
+    
+    <div class="primary-text">
+    ${desktopProject.information}
+    </div>
+    <div class="languages">
+      <ul class="programming-languages">
+      ${language}
+      </ul>
+    </div>
+    <div class="text-see-project">
+      <a class="see-project-button" id=${desktopIndex} href="#">See Project</a>
+    </div>
+  </div>
+</div>
+  </div>
+`;
+  projectContainer.innerHTML += eachDesktopProject;
+});
+
+const projectButtons = document.querySelectorAll('.text-see-project');
+
+const modalContent = (project) => {
+  const popup = `
```

Comment:
- [ ] The popup should be centered as it is in the figma. This is a styling issue though.
### Your work
![image](https://user-images.githubusercontent.com/86200356/210724110-204edd78-c25a-4ab8-9eae-3c1c9234ae62.png)

### Figma
![image](https://user-images.githubusercontent.com/86200356/210724197-82b43170-6929-4a31-ae94-ce24d182f28d.png)